### DISCUSSION TITLE
Metamask not in the list of wallets that can be connected

### DISCUSSION THREAD

**Discussion by 0xtchandragupta** [WalletConnect/walletconnect-monorepo#2628] [Developer Support]
I am using react native web3modal from here https://docs.walletconnect.com/2.0/reactnative/web3modal/installation to connect to an external wallet.

The pop up that shows up does not have metamask.
Is connecting to metamask not supported?
<img width="1430" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/94593328/208293d5-da65-4567-8471-da3bc95470e6">

**Comment by glitch-txs** [comment]
Not supported at the moment, we are waiting for MetaMask upgrade :)

**Comment by jeffohbabygames** [comment]
we are all waiting for metamask :) specifically expo

**Comment by duybuikardiachain** [comment]
Today is their deadline that they committed
Can't wait to see Metamask support WC2
